docs/quick-start.md

```diff
@@ -22,42 +23,41 @@ import {
 ```
 
 ## 3. Get some _AE_ using the Faucet
-To receive some _AE_ you can use the [Faucet](https://faucet.aepps.com/). Just add your publicKey, and you'll immediately get some test coins.
+To receive some _AE_ you can use the [Faucet](https://faucet.aepps.com/). Just paste sender's publicKey, hit `Top UP` and you'll immediately get some test coins.
 
 ## 4. Interact with the æternity blockchain
 This example shows:
 
 - how to create an instance of the SDK using the `Universal` [Stamp](https://stampit.js.org/essentials/what-is-a-stamp)
-- how to get the current block height
-- how to spend 1 AE from the account the SDK instance was initialized with to some other AE address
+- how to spend (send) 1 AE from the account the SDK instance was initialized with to some other AE address
 
 ```js
 const NODE_URL = 'https://testnet.aeternity.io';
-const COMPILER_URL = 'https://compiler.aepps.com'; // required for contract interactions (compiling and creating calldata)
-const account = MemoryAccount({
-  // replace <YOUR_SECRET> and <YOUR_PUBLIC_KEY> with the values from step 2
-  keypair: { secretKey: '<YOUR_SECRET>', publicKey: '<YOUR_PUBLIC_KEY>' }
+const COMPILER_URL = 'https://compiler.aepps.com'; // required for contract interactions
+const senderAccount = MemoryAccount({
+  // replace <SENDER_SECRET> and <SENDER_PUBLIC_KEY> with the generated keyppair from step 2
+  keypair: { secretKey: '<SENDER_SECRET>', publicKey: '<SENDER_PUBLIC_KEY>' }
 })
```

Comment:
Does it works without semicolon here?

Reply:
I think so. Removed

Reply:
I mean it actually needed at the end of statement before a function defined and called at the same time:
```js
console.log('test')
(() => {})()
```
would fail with
> TypeError: console.log(...) is not a function

I've added this while rebasing